cmd/cone/form_fields.go

```diff
@@ -0,0 +1,405 @@
+		return collectBoolField(ctx, field.BoolField, displayName, description)
+	case field.Int64Field != nil:
+		return collectInt64Field(ctx, field.Int64Field, displayName, description)
+	case field.StringSliceField != nil:
+		return collectStringSliceField(ctx, field.StringSliceField, displayName, description)
+	default:
+		return nil, fmt.Errorf("unsupported field type for field: %s", displayName)
+	}
+}
+
+// collectStringField collects a string field value with validation.
+func collectStringField(ctx context.Context, field *shared.StringField, displayName, description string) (string, error) {
+	validator := StringFieldValidator{
+		field:       field,
+		displayName: displayName,
+		description: description,
+	}
+
+	defaultValue := ""
+	if field.DefaultValue != nil {
+		defaultValue = *field.DefaultValue
+	}
+
+	value, err := output.GetValidInput(ctx, defaultValue, validator)
+	if err != nil {
+		return "", err
+	}
+
+	return value, nil
+}
+
+// collectBoolField collects a boolean field value.
+func collectBoolField(ctx context.Context, field *shared.BoolField, displayName, description string) (bool, error) {
+	if description != "" {
+		pterm.Info.Println(description)
+	}
+
+	prompt := fmt.Sprintf("Enter value for '%s' (true/false)", displayName)
+	if field.DefaultValue != nil {
+		prompt = fmt.Sprintf("Enter value for '%s' (true/false, default: %v)", displayName, *field.DefaultValue)
+	}
+
+	result, err := pterm.DefaultInteractiveConfirm.Show(prompt)
+	if err != nil {
+		return false, err
+	}
+
+	return result, nil
+}
+
+// collectInt64Field collects an int64 field value with validation.
+func collectInt64Field(ctx context.Context, field *shared.Int64Field, displayName, description string) (int64, error) {
+	validator := Int64FieldValidator{
+		field:       field,
+		displayName: displayName,
+		description: description,
+	}
+
+	defaultValue := ""
+	if field.DefaultValue != nil {
+		defaultValue = strconv.FormatInt(*field.DefaultValue, 10)
+	}
+
+	value, err := output.GetValidInput(ctx, defaultValue, validator)
+	if err != nil {
+		return 0, err
+	}
+
+	return value, nil
+}
+
+// collectStringSliceField collects a string slice field value.
+func collectStringSliceField(ctx context.Context, field *shared.StringSliceField, displayName, description string) ([]string, error) {
+	if description != "" {
+		pterm.Info.Println(description)
+	}
+
+	prompt := fmt.Sprintf("Enter values for '%s' (comma-separated)", displayName)
+	if len(field.DefaultValues) > 0 {
+		prompt = fmt.Sprintf("Enter values for '%s' (comma-separated, default: %s)", displayName, strings.Join(field.DefaultValues, ", "))
+	}
+
+	userInput := pterm.DefaultInteractiveTextInput.WithMultiLine(false)
+	input, err := userInput.Show(prompt)
+	if err != nil {
+		return nil, err
+	}
+
+	if input == "" {
+		if len(field.DefaultValues) > 0 {
+			return field.DefaultValues, nil
+		}
+		return []string{}, nil
+	}
+
+	// Split by comma and trim whitespace
+	values := strings.Split(input, ",")
```

Comment:
nit: i dont love this part.  wish we could have a multi-entry input loop, press enter with no data to stop adding new rows?  thoughts?